metabench.cmake

```diff
@@ -274,11 +274,11 @@ file(WRITE ${METABENCH_RB_PATH}
 ################################################################################
 set(MEASURE_RB_PATH ${METABENCH_DIR}/measure.rb)
 file(WRITE ${MEASURE_RB_PATH}
-"require 'benchmark'                                                    \n"
-"command = ARGV.join(' ')                                               \n"
-"time = Benchmark.realtime { `#{command}` }                             \n"
-"puts \"[command line: #{command}]\"                                    \n"
-"puts \"[compilation time: #{time}]\"                                   \n"
+"require 'benchmark'                                                         \n"
+"command = ARGV.map{ |arg| arg.match(/\\s/) ? '\"'+arg+'\"' : arg }.join(' ')\n"
```

Comment:
> Nitpick: can you add a space between map and {? 

Sure.

> Also, I can't understand why this works but your previous attempt didn't

A side effect of quoting only when there are spaces is that we avoid quoting things like compiler options that are never expected to be quoted (I guess).